docs/eventing/channels/README.md

```diff
@@ -10,8 +10,6 @@ procedure:
     ```
 -->
 
-# Knative Channels
```

Comment:
Should Channels be capitalized here? I'm seeing a mix of capitalization and non-capitalization - not sure I have strong opinions either way, but I do think it should be consistent.

Reply:
no i dont think so, fixed it below